### PR TITLE
Proposal

### PR DESCRIPTION
**Pull request by michellescripts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+- Meteor?
+- Spotify API
+- Concert API (songkick)
 
 ## Feature list
-
-
+- Playlist based on location
+- Voting system
+- Save concerts based on upvoted songs
```
Comment (peterjostiguy) [inline README.md:40]: I have a feeling your feature list will grow as you work on this, but this is a great place to start

File: README.md [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
-
+- location
+- upvote or downvote songs
 
 ## What technologies do you plan to use?
-
+- React Native
+- Meteor?
+- Spotify API
+- Concert API (songkick)
```
Comment (peterjostiguy) [inline README.md:36]: Remember to include the basics as well- HTML, CSS, etc. They're old hat by now, but they're still the foundation of most apps

**Comment by peterjostiguy** [comment]
Take a look at a couple comments, but no need to resubmit- just feedback for future proposals! Approved!

File: README.md (hunk 2 above)
Comment (michellescripts) [reply README.md:36]: We're not using html and css. We're magic.

Comment (peterjostiguy) [reply README.md:36]: Oh you fancy, huh?

**Pull request closed without merging**